CHANGELOG.next.asciidoc

```diff
@@ -373,6 +373,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Fix retrieving resources by ID for the azure module. {pull}21711[21711] {issue}21707[21707]
 - Use timestamp from CloudWatch API when creating events. {pull}21498[21498]
 - Report the correct windows events for system/filesystem {pull}21758[21758]
+- Stop storing stateless kubernetes keystores {pull}21880[21880]
```

Comment:
Sorry, one last minute suggestion, the changelog should be more user oriented and make reference to the issue fixed.
```suggestion
- Fix panic in kubernetes autodiscover related to keystores {issue}21843[21843] {pull}21880[21880]
```